draft-gharris-opsawg-pcap.md

```diff
@@ -295,6 +291,8 @@ DLT values are associated with specific operation system captures, and are opera
 |LINKTYPE_SCCP|142|SS7 Control Part, ITU-T Q.711/Q.712/Q.713/Q.714
 |LINKTYPE_DOCSIS|143|DOCSIS MAC frames, DOCSIS 3.1
 |LINKTYPE_LINUX_IRDA|144|Linux-IrDA packets w/LINKTYPE_LINUX_IRDA header
+|LINKTYPE_PRIVATE_USE_1|147|For private use
```

Comment:
There aren't 2 private use values, there are 16 private use values, 147 to 162, so they'd be `LINKTYPE_PRIVATE_USE_1` (`LINKTYPE_USER1`) through `LINKTYPE_PRIVATE_USE_16` (`LINKTYPE_USER16`).